File: lib/Ntreev.Windows.Forms.Grid/GridRow.cpp
#include "stdafx.h"
#include "GridRow.h"
#include "NativeGridRow.h"

namespace Ntreev { namespace Windows { namespace Forms { namespace Grid
{
    GridRow::GridRow(Native::GrGridRow* pGridRow)
        : m_pGridRow(pGridRow), RowBase(pGridRow)
    {
    }

    Ntreev::Windows::Forms::Grid::GridControl^ GridRow::ChildGrid::get()
    {
        return m_pGridRow->GetChildGrid();
    }


} /*namespace Grid*/ } /*namespace Forms*/ } /*namespace Windows*/ } /*namespace Ntreev*/

File: lib/WinformGridHost/GridRow.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Ntreev.Windows.Forms.Grid
{
    public class GridRow : RowBase
    {
        private GrGridRow m_pGridRow;

        internal GridRow(GrGridRow pGridRow)
            : base(pGridRow)
        {

        }

        public GridControl ChildGrid
        {
            get { return m_pGridRow.GetChildGrid(); }
        }
    }
}
